src/include/cluster/dialogs.rb

```diff
@@ -708,6 +708,205 @@ def SaveSecurity
       nil
     end
 
+    def ValidateCorosyncQdevice
+      if UI.QueryWidget(Id(:corosync_qdevice), :Value) == false
+        return true
+      end
+
+      if UI.QueryWidget(Id(:qdevice_model), :Value) != "net"
+        Popup.Message(_("The model basically defines the type of arbitrator, currently only net is support"))
+        UI.SetFocus(:qdevice_model)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_votes), :Value).to_i <= 0
+        Popup.Message(_("Qdevice votes must be a positive integer"))
+        UI.SetFocus(:qdevice_votes)
+        return false
+      end
+
+      if !IP.Check(UI.QueryWidget(Id(:qdevice_host), :Value))
+        Popup.Message(_("Qdevice host mush have a valid IP address"))
+        UI.SetFocus(:qdevice_host)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_port), :Value).to_i <= 0
+        Popup.Message(_("The corosync qdevice port must be a positive integer"))
+        UI.SetFocus(Id(:qdevice_port))
+        return false
+      end
+
+      if !["ffsplit", "lms", "test", "2nodelms"].include?(UI.QueryWidget(Id(:qdevice_algorithm), :Value))
+        Popup.Message(_("The algorithm only can be one of the ffsplit, lms, test or 2nodelms.
+          YaST will overwrite test and 2nodelms."))
+        UI.SetFocus(Id(:algorithm))
+        return false
+      end
+
+      if !["lowest", "highest"].include?(UI.QueryWidget(Id(:qdevice_tie_breaker), :Value)) &&
+        (UI.QueryWidget(Id(:tie_breaker), :Value).to_i <= 0)
+        Popup.Message(_("The tie breaker can be one of lowest, highest or a valid node id (number)"))
```

Comment:
I am a bit confused here, widget is `qdevice_tie_breaker` or `tie_breaker`? you query both.

Reply:
oops, it is a bug. I changed the label to :qdevice_tie_breaker but missing two of them.
Thanks for the catch.